Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  FairmaticSDKTargetPlaceholder.swift
+//  
+//
+//  Created by Sagar Dagdu on 02/05/24.
```

Comment:
we usually omit the Created by header in public files. we can keep Fairmatic copyright header instead

Reply:
That is correct, will change this

Reply:
Done